variables.tf

```
@@ -1,4 +1,122 @@
-variable "example" {
-  description = "Example variable"
-  default     = "hello world"
+variable "node_type" {
+  description = "The compute and memory capacity of the nodes in the cluster"
+  type        = string
+  default     = "db.r6g.large"
+}
```

Add `nullable = false` to all variables with defaults, unless the default is `null`. For example:

```suggestion
variable "node_type" {
  description = "The compute and memory capacity of the nodes in the cluster"
  type        = string
  default     = "db.r6g.large"
  nullable    = false
}
```